src/styles/blocks/card.scss

```
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: $size-template;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: all $time-activ;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%;
+    margin: 0 0 4px;
+    color: $titleColor;
+  }
+
+  &:hover &__title {
+    transition: color $time-activ;
+    color: $text-color-hover;
+  }
+
+  &__product-code {
+    color: $productColor;
+    font-size: 10px;
+  }
+
+  &__reviews {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 17px;
+  }
+
+  &__reviews-text {
+    font-size: 10px;
+    line-height: 14px;
+  }
```

Let's fix it everywhere in styles
```suggestion
  &__reviews {
    width: 100%;
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-top: 17px;
    
    &-text {
      font-size: 10px;
      line-height: 14px;
    }
  }
```